app/channels/queue_channel.rb

```diff
@@ -138,6 +138,21 @@ def destroy_request(data)
     broadcast_request_change('resolve_request', request)
   end
 
+  def merge(data)
+    authorize :instructor_only
+    @to_course_queue = CourseQueue.find_by!(id: data['to'])
```

Comment:
We should also make sure the user is an instructor on `@to_course_queue`

Reply:
Done